public/admin-order-detail.js

```diff
@@ -30,6 +30,52 @@
 		$( '[data-packetery-label-print-modal] .modal-close:visible:first' ).click();
 	} );
 
-	new PacketeryMultiplier('[data-packetery-customs-declaration-item]');
+	$( 'body' ).on( 'wc_backbone_modal_loaded', function( e ) {
```

Comment:
V tomto nově přidaném kódu není jednotný code style.

Reply:
Upraveno.

Reply:
Prosím aplikovat formátování na nově přidanou část.
Upravil jsi code style ve staré části, to je potřeba vrátit, nebo si to domluvit.

Reply:
Formatovani aplikovano.